integration-tests/utils/integration-utils.js

```diff
@@ -497,6 +522,46 @@ const verifyAccessibility = async (page, allowedViolations) => {
   }
 };
 
+/** Converts various time units to milliseconds. */
+const Millis = (() => {
```

Comment:
Nit: I'd vote for not abbreviating this. The core feature of this is that it's representing a duration of time. `Millis` alone sounds like it's a generic metric prefix utility.

```suggestion
const Milliseconds = (() => {
```

Reply:
Here I was taking Java's time libraries' example, where 'millis' is a standard abbreviation for milliseconds - like `System.currentTimeMillis()`  and `toEpochMilli()`.